framework/src/modules/reward/calculate_reward.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2020 Lisk Foundation
```

Comment:
```suggestion
 * Copyright © 2021 Lisk Foundation
```
Ignore this comment if this file was only moved

Reply:
It wasn't moved, need to fix it 😄

Reply:
done